source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-tuning-pid.rst

```diff
@@ -28,6 +28,34 @@ This is the test mode picture of a wrist subsystem that has a potentiometer as t
 4. The setpoint value that corresponds the to the pot value when the wrist has reached the desired value
 5. Enables the PID controller - that is, starts it looping at regular intervals reading the pot value, computing the error, applying the P, I, and D terms and setting the motor value.
 
-.. important:: The enable option does not affect the `PIDController <https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/controller/PIDController.html>`__ introduced in 2020, as the controller is updated every robot loop. Users who wish to retain this functionality can retrieve the value from NetworkTables and implement the feature themselves.
+.. important:: The enable option does not affect the `PIDController <https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/controller/PIDController.html>`__ introduced in 2020, as the controller is updated every robot loop. See the example below on how to retain this fuctionality.
+
+Implementing Enable Functionality in the New PIDController
```

Comment:
This title is particularly long, and doesn't wrap well on mobile. Can we shorten this?

Reply:
what about "Enable Functionality in the New PIDController" or is that still too long?